CHANGELOG.md

```diff
@@ -80,6 +58,71 @@ Check your changes against the [header example in the Design System](https://des
 
 This change was introduced in [pull request #2427: Improve the screen reader experience for the header menu](https://github.com/alphagov/govuk-frontend/pull/2427).
```

Comment:
Do we need to mention [#2471: Add missing instructions for header to changelog](https://github.com/alphagov/govuk-frontend/pull/2471)?

Reply:
@36degrees @vanitabarrett

Reply:
No need to mention this, will delete this comment.